src/controllers/helpers/pagination.rs

```diff
@@ -411,7 +411,7 @@ mod tests {
     }
 
     fn mock(query: &str) -> MockRequest {
-        let mut req = MockRequest::new(http::Method::GET, "");
+        let mut req = MockRequest::new(http::Method::GET, "/");
```

Comment:
an empty `URI` is invalid, so we had to change the default `path` value here to make the tests pass without panicking